src/index.html

```diff
@@ -13,6 +13,47 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars--0">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
```

Comment:
The class names should represent the meaning of the content, not the styles or tag names. Instead of 'stars--0', 'stars--1', etc, you can use more semantic names like 'stars--none', 'stars--one', etc. Also, instead of 'stars__star', you can use 'star' or 'star--active' and 'star--inactive' to represent the state of each star.